.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
 ###### Description
 
 
+<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->
```

Comment:
I have some problems understand what exactly I'm supposed to do here to make this work. Just use
```
[skip notification]
```
on a line on its own?

Should this be uncommented or may I just modify the existing sentence inside the comment?